application/phoenix/README.md

```diff
@@ -0,0 +1,35 @@
+This Phoenix example checks if the environment variables `DEMO_USERNAME` and `DEMO_PASSWORD` have been set. If that's the case, you'll receive a personalized greeting message on http://localhost:4000, otherwise you will get a "Hello stranger" message.
+
+## Prerequisites
+1. [Docker](https://docs.docker.com/install/) installed and running
+1. [SecretHub](https://secrethub.io/docs/start/getting-started/#install) installed
+1. A SecretHub repo that contains a `username` and `password` secret. To create it, run `secrethub demo init`.
+
+## Running the example
+
+Set the SecretHub username in an environment variable
+```
+export SECRETHUB_USERNAME=<your-username>
+```
+
+Create a service account for the demo repo
+```
+secrethub service init --description demo_service \
+--permission read --file demo_service.cred ${SECRETHUB_USERNAME}/demo
+```
+
+Build the Phoenix docker demo
+```
+docker build . -t phoenix-secrethub-demo
+```
+
+Run the docker demo with the secrets in the environment variables
+```
+docker run -p 8080:8080 \
```

Comment:
https://github.com/secrethub/examples/blob/d66fc06d6af69805b8ce8403403b1fe97287d20c/application/phoenix/Dockerfile#L15

In the Dockerfile you exposed port `4000` but here you map port `8080` to port `8080`. This is a mistake, right?

Reply:
Yeppers, I'm fixing this now.